.github/workflows/update_weekly_rosters.yaml

```diff
@@ -0,0 +1,44 @@
+on:
+  schedule:
+    # runs every day at 7:00 AM UTC
+    - cron:  '0 7 * * *'
+  # push:
+    # paths:
+      # - R/update_roster.R
+      #- R/update-injuries.R
+  workflow_dispatch:
+
+name: Update-Rosters-Weekly
+
+jobs:
+  update-rosters-weekly:
+    name: Update-Rosters-Weekly
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.NFLVERSE_GH_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
+            nflverse/nflreadr
+            nflverse/nflverse-data
+            nflverse/ngsscrapR
+
+      - name: Update weekly rosters
+        run: Rscript -e 'source("src/update-weekly-rosters.R")'
+
+      - name: Commit and push roster data
```

Comment:
should not commit and push at all (data only stored on nflverse-data)

this is probably a leftover from the previous roster action which is wrong